femto/fe/septop/_setup.py

```diff
@@ -241,12 +241,14 @@ def setup_complex(
         )
         receptor_ref_idxs_2 = receptor_ref_idxs_1
 
+        # Remove the offset and check use ligand 2 indices with ligand 2.
```

Comment:
Could you please reformulate the comment for clarity? For example: Remove the offset, ligand_2 reference indices should be in range 0:ligand_2.n_atoms to match ligand_2 parmed.Structure.

Reply:
No problem. Done.